packages/core/addon/components/navi-visualizations/goal-gauge.ts

```diff
@@ -139,38 +129,27 @@ export default class GoalGaugeVisualization extends Component<Args> {
     };
   }
 
-  /**
-   * @property {Array} - colors to render corresponding to the thresholdValues
-   */
-  @alias('config.thresholdColors') thresholdColors = ['#f05050', '#ffc831', '#44b876'];
-
-  /**
-   * @property {Array} - percentages to render corresponding to the colors
-   */
-  @alias('config.thresholdPercentages') thresholdPercentages = [75, 85, 100];
-
   /**
    * @property {Array} - threshold values to indicate what color to render
    * C3 Threshold Algorithm: if < threshold value indexN, use color indexN
    */
   @computed('args.options.{baselineValue,goalValue}')
```

Comment:
update computed keys

Reply:
This isnt showing as outdated but its not in the file anymore